Formula/qdated.rb

```diff
@@ -0,0 +1,42 @@
+class Qdated < Formula
+  desc "Create and verify timestamped email addresses for qmail"
+  homepage "https://web.archive.org/web/http://www.palomine.net/qdated/"
+  url "https://web.archive.org/web/http://www.palomine.net/qdated/qdated-0.53.tar.gz"
+  sha256 "002391c656c705838965cab8231261f2474b26711c03e24cf0ecf2978b3cf934"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
+
+  def install
+    Dir.chdir("qdated-#{version}") do
+      system "package/compile"
+    end
+
+    bin.install "qdated-#{version}/command/qdated-check"
+    bin.install "qdated-#{version}/command/qdated-makekey"
+    bin.install "qdated-#{version}/command/qdated-now"
+
+    File.rename "qdated-#{version}/package/LICENSE", "LICENSE"
+  end
+
+  test do
+    require "English"
```

Comment:
Why is this required?

Reply:
The test uses the `$CHILD_STATUS` variable, which is provided by the `English` library. I was using `$?` before, but `brew audit` told me to use `$CHILD_STATUS` from `English` instead.